lte/gateway/python/magma/enodebd/data_models/transform_for_magma.py

```diff
@@ -65,18 +65,20 @@ def bandwidth(bandwidth_rbs: Union[str, int, float]) -> float:
     BaiCells eNodeB uses 'n6'. Need to resolve this.
 
     Args:
-        bandwidth_rbs (str): Bandwidth in number of RBs
+        bandwidth_rbs (str, float): Bandwidth in number of RBs
     Returns:
         str: Bandwidth in MHz
     """
-    if bandwidth_rbs in BANDWIDTH_RBS_TO_MHZ_MAP:
+    if isinstance(bandwidth_rbs, str) and \
+            bandwidth_rbs in BANDWIDTH_RBS_TO_MHZ_MAP:
         return BANDWIDTH_RBS_TO_MHZ_MAP[bandwidth_rbs]
 
     logger.warning('Unknown bandwidth_rbs (%s)', str(bandwidth_rbs))
-    if bandwidth_rbs in BANDWIDTH_MHZ_LIST:
+    if isinstance(bandwidth_rbs, (int, float)) and \
+            bandwidth_rbs in BANDWIDTH_MHZ_LIST:
         return bandwidth_rbs
     elif isinstance(bandwidth_rbs, str):
-        mhz = None
+        mhz: Union[int, float, None] = None
```

Comment:
I think this could just be an `Optional[float]`.

`int` is treated like a subtype of `float` so if you have `float` you don't need a union with `int` in general. See: https://mypy.readthedocs.io/en/latest/duck_type_compatibility.html